README.md

```diff
@@ -14,7 +14,7 @@ The package can be installed by adding `membrane_rtmp_plugin` to your list of de
 ```elixir
 def deps do
   [
-	  {:membrane_rtmp_plugin, "~> 0.14.0"}
+	  {:membrane_rtmp_plugin, "~> 0.14.1"}
```

Comment:
What do you think about bumping it to `0.15.0` and getting rid of `finalize_stream`?

Reply:
I wouldn't break the public API because I think
- `finalize_stream` is OKish, I find it intuitive
- user will get the benefit of the new implementation w/o code changes
- I see this more as a bugfix rather than a feature update

Said that, we can get rid of that function if you prefer to!

Reply:
Ok, seems fair enough :)